### ISSUE TITLE
"#EXTVLCOPT:http-user-agent=" function broken in 0.4.5.2

### ISSUE THREAD

**Issue by VirgilClyne** [stari4ek/tvirl#26]
"#EXTVLCOPT:http-user-agent=" function broken in 0.4.5.2

**Comment by stari4ek** [comment]
Thank you for reporting.

There is an issue, that 2 UA entries sent. So, "last wins".

```
#EXTINF:-1, HLS: Apple 4x3 basic stream
#EXTVLCOPT:http-user-agent=Some User Agent
https://devstreaming-cdn.apple.com/videos/streaming/examples/bipbop_4x3/bipbop_4x3_variant.m3u8
```

```
2020-05-10 14:57:41.844 2908-3096/by.stari4ek.tvirl.debug D/HttpLoggingInterceptor: Sending GET request to https://devstreaming-cdn.apple.com/videos/streaming/examples/bipbop_4x3/bipbop_4x3_variant.m3u8
    User-Agent: Some User Agent
    User-Agent: TVirl/0.4.5.2-dirty-dev (Linux;Android 10) ExoPlayerLib/2.11.4
    Host: devstreaming-cdn.apple.com
    Connection: Keep-Alive
    Accept-Encoding: gzip

2020-05-10 14:57:41.919 2908-3098/by.stari4ek.tvirl.debug D/HttpLoggingInterceptor: Sending GET request to https://devstreaming-cdn.apple.com/videos/streaming/examples/bipbop_4x3/gear1/prog_index.m3u8
    User-Agent: Some User Agent
    User-Agent: TVirl/0.4.5.2-dirty-dev (Linux;Android 10) ExoPlayerLib/2.11.4
    Host: devstreaming-cdn.apple.com
    Connection: Keep-Alive
    Accept-Encoding: gzip

2020-05-10 14:57:42.107 2908-3099/by.stari4ek.tvirl.debug D/HttpLoggingInterceptor: Sending GET request to https://devstreaming-cdn.apple.com/videos/streaming/examples/bipbop_4x3/gear1/fileSequence0.ts
    User-Agent: Some User Agent
    User-Agent: TVirl/0.4.5.2-dirty-dev (Linux;Android 10) ExoPlayerLib/2.11.4
    Accept-Encoding: identity
    Host: devstreaming-cdn.apple.com
    Connection: Keep-Alive
```

Will work on hot fix for `0.4.5.2`

**Comment by stari4ek** [comment]
Fixed as part of 0.4.5.3
